components/delegations/cells/ControlsCell.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import Button from 'components/common/Button';
+import type { Delegation } from 'lib/delegate/DelegatePlatform';
+import { useRevokeDelegation } from 'lib/hooks/ethereum/useRevokeDelegation';
+import { useTranslations } from 'next-intl';
+
+interface Props {
+  delegation: Delegation;
+  onRevoke: (delegation: Delegation) => void;
+}
+
+const ControlsCell = ({ delegation, onRevoke }: Props) => {
```

Comment:
I recommend structuring this similarly to the `ControlsCell` for sessions with the `ControlsWrapper`, so that is inherits that functionality.